package.json

```diff
@@ -36,6 +36,7 @@
     "build": "node scripts/babelcheck.js && npm run build:res && npm run build:bundle",
     "build:dev": "node scripts/babelcheck.js && npm run build:res && npm run build:bundle:dev",
     "dist": "scripts/package.sh",
+    "postinstall": "cd electron && npm i",
```

Comment:
this looks weird - we surely don't want to try to install all the electron crap every time someone runs `npm i` on riot-web?